openlibrary/core/models.py

```diff
@@ -827,7 +827,7 @@ def has_librarian_tools(self):
     def is_read_only(self):
         return self.is_usergroup_member('/usergroup/read-only')
 
-    def get_lists(self, seed=None, limit=100, offset=0, sort=True):
+    def get_lists(self, seed=None, limit=1000, offset=0, sort=True):
         """Returns all the lists of this user.
 
         When seed is specified, this returns all the lists which contain the
```

Comment:
Perhaps line 839 should be modified as well?

Reply:
Makes sense. I should have done that. Thank you!

Reply:
@cclauss Good point! I also went ahead and changed whether it is cached or not. Thank you!